sycl/include/sycl/known_identity.hpp

```diff
@@ -258,10 +258,24 @@ template <typename BinaryOperation, typename AccumulatorT>
 struct known_identity_impl<BinaryOperation, AccumulatorT,
                            std::enable_if_t<IsMinimumIdentityOp<
                                AccumulatorT, BinaryOperation>::value>> {
+// TODO: detect -fno-honor-infinities instead of -ffast-math
```

Comment:
Should we create a GH/internal JIRA to track this TODO?

Reply:
have already done so. CMPLRLLVM-65507